src/main/resources/hudson/plugins/logparser/LogParserPublisher/config.jelly

```diff
@@ -9,7 +9,7 @@
   <f:entry title="Show log parser graphs" help="/plugin/log-parser/parser_graphs.html">
     <f:checkbox name="log-parser.showGraphs" checked="${instance.showGraphs}"/>
   </f:entry>
-  <f:radioBlock title="Use global rule" name="log-parser.useProjectRule" value="false" checked="${instance.useProjectRule!=true}">
+  <f:radioBlock title="Use global rule" name="log-parser.useProjectRule" value="false" checked="${instance.useProjectRule!=true}" inline="true">
```

Comment:
@amuniz It seems [there aren't](https://github.com/jenkinsci/acceptance-test-harness/search?utf8=%E2%9C%93&q=LogParser) acceptance test for this plugin.